backend/app/models/company.rb

```diff
@@ -120,6 +120,8 @@ def deactivate! = update!(deactivated_at: Time.current)
 
   def active? = deactivated_at.nil?
 
+  def primary_admin = super || company_administrators.first
```

Comment:
To be consistent with the previous behavior:

```suggestion
  def primary_admin = super || company_administrators.order(:id).first
```